src/test/java/io/codurance/pouch/ResourceFavouriteControllerShould.java

```diff
@@ -0,0 +1,62 @@
+package io.codurance.pouch;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static java.time.Instant.now;
+import static java.util.UUID.randomUUID;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.http.HttpStatus.valueOf;
+
+class ResourceFavouriteControllerShould {
+
+    private ResourceRepository ResourceRepository;
```

Comment:
I'd rename the member with a lowercase 'r' and consider making members final?

Reply:
Yes on the renaming, same as above. About making the fields final...that would require me to move the assignment to the field declaration, instead of using the `setUp()` method for initialization. I'll go with the `setUp()` method for now, as I am using this pattern frequently.